src/services/pure-di.tsx

```diff
@@ -131,6 +133,18 @@ export class AppCompositionRoot implements AppServices {
     );
   }
 
+  get dopplerAPIClient() {
```

Comment:
What do you think about using the c# convention of using uppercase only for the first letter of acronyms with more than two letters?

Reply:
```suggestion
  get dopplerApiClient() {
```